generators/ansible-do/templates/ansible/stack/makefile

```diff
@@ -22,7 +22,7 @@ docker.data-folder:
 
 install:
 	@printf "\033[0;32m>>> Install ansible roles\033[0m\n"
-	ansible-galaxy install -r requirements.yml --force
+	ansible-galaxy collection install dzangolab.ansible --force
```

Comment:
collections can be listed in requirements.txt. See https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html#collection-requirements-file. This is ultimately more flexible than hard-coding the collection to install in the makefile. (the makefile is only a convenience tool, it should not contain any logic)